cadctl/cmd/investigate/investigate.go

```diff
@@ -222,12 +223,17 @@ func isAWS(cluster *cmv1.Cluster) (bool, error) {
 	return cloudProvider.ID() == "aws", nil
 }
 
+// ClusterDeploymentSupportExceptionLabel is the label indicating the cluster is under limited support
+// and the PagerDuty service should be enabled even if the cluster is in limited support
+// TODO:() Centrailize or Remove this varaiable once https://issues.redhat.com/browse/XCMSTRAT-427 is completed
+var ClusterDeploymentSupportExceptionLabel string = "hive.openshift.com/support-exception"
```

Comment:
Why would we tackle this in CAD instead of the pagerduty operator layer (which is the component disabling services when a cluster is in LS)? If we want to make sure support exception clusters alert SRE, we should tackle it on a higher level than the alert receiving end (e.g. CAD).

Also, we are planning to revert the workaround that results in the DMS service not being disabled during limited support, meaning after that alerts for clusters that are in LS should not even make it to CAD.

Reply:
@typeid there is a customer support related issue that is related to an incident that occurred this is a BU temporary hot-fix until we are able to address it post holiday

Reply:
Gotcha. Is there a specific reason we tackle this only in CAD though? Did the cluster trigger a CHGM that was skipped due to LS?
What's with all the other cluster related alerts that are not CHGM?

Reply:
Ah, I see there's https://github.com/openshift/pagerduty-operator/pull/253. Nevermind the question then :)